hack/test-end-to-end-docker.sh

```diff
@@ -65,8 +65,7 @@ function cleanup()
 	exit $out
 }
 
-trap "exit" INT TERM
-trap "cleanup" EXIT
+trap "cleanup" EXIT INT TERM
```

Comment:
> There are no containers to be logged but there is a record of us logging them. I guess we could break it earlier and see the logs and I'll do that but we did enter the container logging function

I see no line: `[FAIL] !!!!! Test Failed !!!!`, which should be the first line from function, right?